File: Theater Ticket Booking/Színház_jegyfoglaló_app/Színház_jegyfoglaló_app/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Data.SQLite;

namespace Színház_jegyfoglaló_app
{
    public partial class Form1 : Form
    {
        DataTable dt = new DataTable();
        SQLiteDataAdapter sda;
        SQLiteCommand cmd = new SQLiteCommand();

        public Form1()
        {
            InitializeComponent();
        }

        private void infoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            MessageBox.Show("Készítette: Kőrösi Krisztofer Roland\nVerziószám: 1.0\nKiadás dátuma: 2020.11.17");
        }

        private void kilépésToolStripMenuItem_Click(object sender, EventArgs e)
        {
            System.Environment.Exit(0);
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.Hide();
            Regisztracio r = new Regisztracio();
            r.Show();
        }

        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            System.Environment.Exit(0);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            if(textBox1.Text == "rendszergazda" && textBox2.Text == "rendszergazda1" && radioButton1.Checked)
            {
                this.Hide();
                Rendszergazda admin = new Rendszergazda();
                admin.Show();
            }
            else if (radioButton2.Checked)
            {
                DB db = new DB();
                cmd = new SQLiteCommand("select count(*) from belepes where felhnev='" + textBox1.Text + "' and jelszo='" + textBox2.Text + "'", db.getConnection());
                sda = new SQLiteDataAdapter(cmd);
                sda.Fill(dt);
                if(dt.Rows[0][0].ToString() == "1")
                {
                    this.Hide();
                    User u = new User();
                    u.Show();
                }
                else
                {
                    MessageBox.Show("Hibás felhasználónév és/vagy jelszó!\nKérjük próbáld újra!");
                }
            }
        }
    }
}

File: Theater Ticket Booking/Színház_jegyfoglaló_app/Színház_jegyfoglaló_app/Form1.vb
Imports System
Imports System.Data
Imports System.Windows.Forms
Imports System.Data.SQLite

Namespace Színház_jegyfoglaló_app
    Public Partial Class Form1
        Inherits Form

        Private dt As DataTable = New DataTable()
        Private sda As SQLiteDataAdapter
        Private cmd As SQLiteCommand = New SQLiteCommand()

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub infoToolStripMenuItem_Click(ByVal sender As Object, ByVal e As EventArgs)
            MessageBox.Show("Készítette: Kőrösi Krisztofer Roland" & Microsoft.VisualBasic.Constants.vbLf & "Verziószám: 1.0" & Microsoft.VisualBasic.Constants.vbLf & "Kiadás dátuma: 2020.11.17")
        End Sub

        Private Sub kilépésToolStripMenuItem_Click(ByVal sender As Object, ByVal e As EventArgs)
            Environment.Exit(0)
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Hide()
            Dim r As Regisztracio = New Regisztracio()
            r.Show()
        End Sub

        Private Sub Form1_FormClosing(ByVal sender As Object, ByVal e As FormClosingEventArgs)
            Environment.Exit(0)
        End Sub

        Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs)
            If Equals(textBox1.Text, "rendszergazda") AndAlso Equals(textBox2.Text, "rendszergazda1") AndAlso radioButton1.Checked Then
                Hide()
                Dim admin As Rendszergazda = New Rendszergazda()
                admin.Show()
            ElseIf radioButton2.Checked Then
                Dim db As DB = New DB()
                cmd = New SQLiteCommand("select count(*) from belepes where felhnev='" & textBox1.Text & "' and jelszo='" & textBox2.Text & "'", db.getConnection())
                sda = New SQLiteDataAdapter(cmd)
                sda.Fill(dt)

                If Equals(dt.Rows(0)(0).ToString(), "1") Then
                    Hide()
                    Dim u As User = New User()
                    u.Show()
                Else
                    MessageBox.Show("Hibás felhasználónév és/vagy jelszó!" & Microsoft.VisualBasic.Constants.vbLf & "Kérjük próbáld újra!")
                End If
            End If
        End Sub
    End Class
End Namespace
